test/testing.cmake

```diff
@@ -298,6 +298,10 @@ mpi_test(tet_serial 1
   "${MDIR}/pipe.${GXT}"
   "pipe.smb"
   "tet.smb")
+mpi_test(test_residual_error_estimate 1
+  ./residualErrorEstimation_test
+  "${MESHES}/electromagnetic/fichera.x_t"
```

Comment:
This needs to be in an `if(ENABLE_SIMMETRIX)` conditional to read the parasolid model.

Reply:
Done.